tools/infrastructure/api_compare.py

```diff
@@ -0,0 +1,368 @@
+#!/usr/bin/env python
+
+import json
+import sys
+import argparse
+import xml.etree.ElementTree as ET
+from collections import namedtuple
+reload(sys)
+
+# Command line arguments parsing
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "-f1", "--mobile_api_file",
```

Comment:
@LuxoftAKutsan It can be. But according coding style all parameters should be located one under other with the indent of 4 spaces from line beginning. This function has 3 parameters.